admin/controllers/group.js

```diff
@@ -0,0 +1,29 @@
+const groupService = require('../services/group.service')
+
+/**
+ * Render the initial 'groups' page.
+ * @param req
+ * @param res
+ * @param next
+ * @returns {Promise<void>}
+ */
+const groupPupilsPage = async (req, res, next) => {
+  res.locals.pageTitle = 'Group pupils'
+
+  let groups
```

Comment:
@GeorgeChatzigiannis I added a try/catch in the service, this page should render anyway even if groups fail.

Reply:
@pris54 this is not a practice we are following somewhere else though. I 'd rather keep it consistent and have the try catch block in the controller rather than the service.
It should be a really simple change.

Reply:
@GeorgeChatzigiannis That's not quite true, go through the controllers and have a look. I'm adding the try/catch at your request but to be clear: I still want to render the page if that promise fails.